docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -805,6 +805,17 @@ Using the Payment Observer allows you to delegate this step to the Anchor Platfo
 
 The Payment Observer will track all transactions sent to the distribution account. When the transaction with the correct memo is detected in the network, the status will automatically change to `pending_anchor` and event will be the emitted (if Kafka is used).
 
+In order to update transaction statuses, the observer makes `PATCH /transactions` requests to the platform, so lets configure the URL is should use.
+
+<CodeExample>
+
+```bash
+# dev.env
+PLATFORM_API_BASE_URL=http://platform-server
```

Comment:
Pretty sure you need to add port too
https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/service-runner/src/main/resources/profiles/sep24/config.env#L19

Reply:
Good catch, fixed!